locale/00_UpdateLocalization.rb

```diff
@@ -0,0 +1,10 @@
+require_relative '../RubySetupSystem/RubyCommon'
+
+puts 'Extracting .pot file'
+runOpen3Checked 'pybabel', 'extract', '-F', 'babelrc', '-k', 'LineEdit', '-k', 'text', '-k', 'window_title', '-k', 'dialog_text', '-k', 'Translate', '-o', 'messages.pot', '../.'
```

Comment:
Do these all flags need to be specified on the command line? Could these be in the babelrc config file?

Reply:
No, they unfortunately can only be passed from the command line

Reply:
That's unfortunate. At least they can be stored in the script so that everyone uses the same flags when running pybabel.